samples/tfm_integration/psa_level_1/README.rst

```diff
@@ -140,13 +140,46 @@ and non-secure (``zephyr.hex``) images wth a J-Link as follows:
    .. code-block:: console
 
       JLinkExe -device lpc55s69 -if swd -speed 2000 -autoconnect 1
-      J-Link>loadfile build/tfm/install/outputs/LPC55S69/tfm_s.hex
+      J-Link>loadfile build/tfm/install/outputs/NXP/LPCXPRESSO55S69/tfm_s.hex
       J-Link>loadfile build/zephyr/zephyr.hex
 
 NOTE: At present, the LPC55S69 doesn't include support for the MCUBoot bootloader.
 
 We need to reset the board manually after flashing the image to run this code.
 
+On nRF5340 and nRF9160:
+=======================
+
+Build Zephyr with a non-secure configuration
+(``-DBOARD=nrf5340pdk_nrf5340_cpuappns`` or ``-DBOARD=nrf9160dk_nrf9160ns``).
+
+   Example, for nRF9160, using ``cmake`` and ``ninja``
+
+   .. code-block:: bash
+
+      cd <ZEPHYR_ROOT>/samples/tfm_integration/psa_level_1/
+      rm -rf build
+      mkdir build && cd build
+      cmake -GNinja -DBOARD=nrf9160dk_nrf9160ns ..
```

Comment:
Just wondering, why not just do:
```suggestion
      cmake -GNinja -DBOARD=nrf9160dk_nrf9160ns -Bbuild .
```
that will create the `build` folder and thus save the `mkdir builld && cd build` step

Note, this is not blocking in this PR, as I know this description has been used elsewhere.

Reply:
Yeap, we sort of use the old description in most of the cases, so while this is a good suggestion, I'd prefer not to implement it for this particular sample (but we could apply tree-wide later)

Reply:
yeap. Fully accepted, just wanted to point it out.